samples/net/azure_iot_hub/boards/native_sim.conf

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# General
+CONFIG_LOG_BUFFER_SIZE=2048
+CONFIG_HW_ID_LIBRARY=y
+CONFIG_ASSERT=y
+CONFIG_FPU=y
+
+# Heap and stacks
+CONFIG_INIT_STACKS=y
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
+
+# Networking config
+CONFIG_NET_IPV6=y
+CONFIG_NET_TCP=y
+CONFIG_NET_UDP=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+CONFIG_NET_PKT_RX_COUNT=500
+CONFIG_NET_PKT_TX_COUNT=500
+CONFIG_NET_BUF_RX_COUNT=600
+CONFIG_NET_BUF_TX_COUNT=600
+CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=3
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=4
+CONFIG_NET_MAX_CONTEXTS=10
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_CONTEXT_SNDTIMEO=y
+
+CONFIG_ETH_NATIVE_POSIX=y
+CONFIG_ETH_NATIVE_POSIX_RANDOM_MAC=y
+
+# Network driver config
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+# TLS configuration
+CONFIG_MBEDTLS=y
+CONFIG_MBEDTLS_BUILTIN=y
+CONFIG_MBEDTLS_ENABLE_HEAP=y
+CONFIG_MBEDTLS_HEAP_SIZE=40000
+CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=7168
+CONFIG_MBEDTLS_PEM_CERTIFICATE_FORMAT=y
+CONFIG_MBEDTLS_KEY_EXCHANGE_RSA_ENABLED=y
+CONFIG_MBEDTLS_MAC_SHA256_ENABLED=y
+
+# DNS
+CONFIG_DNS_RESOLVER=y
+CONFIG_DNS_SERVER_IP_ADDRESSES=y
+CONFIG_DNS_SERVER1="8.8.8.8"
+CONFIG_DNS_SERVER2="1.1.1.1"
+CONFIG_DNS_RESOLVER_MAX_SERVERS=2
+
+# Network address config
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_NEED_IPV4=y
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.0.2.1"
+CONFIG_NET_CONFIG_PEER_IPV4_ADDR="192.0.2.2"
+CONFIG_NET_CONFIG_MY_IPV4_GW="192.0.2.2"
+
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_NATIVE_POSIX_SLOWDOWN_TO_REAL_TIME=y
+
+# Azure IoT Hub library
+CONFIG_MQTT_HELPER_SEC_TAG=201
+CONFIG_MQTT_HELPER_SECONDARY_SEC_TAG=-1
+CONFIG_MQTT_HELPER_STACK_SIZE=2560
```

Comment:
Why do we need to bump the stack size of this library for this sample but not the others?

Reply:
not needed, I'll remove